### PR TITLE
Added solution to SPR-05

### PR DESCRIPTION
**Pull request by MichaelShevchenko**

### CODE REVIEW

**Review by dariiabutorina — APPROVED** [review]
Looks good

File: src/main/java/com/dev/cinema/controller/MovieController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movies")
+public class MovieController {
+    private final MovieService movieService;
+    private final MovieMapper movieMapper;
+
```
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/controller/MovieController.java:22]: if you use Autowired on AuthenticationController, put it on each constructor to keep code consistent

File: src/main/java/com/dev/cinema/controller/MovieController.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        movieService.add(new Movie("Fearless", "Martial arts"));
+        return "Data added successfully";
+    }
+
+    @PostMapping
+    public void add(@RequestBody MovieRequestDto movieDto) {
+        movieService.add(movieMapper.convertDtoToMovie(movieDto));
+    }
+
+    @RequestMapping(method = RequestMethod.GET)
```
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/controller/MovieController.java]: Why not GetMapping?

File: src/main/java/com/dev/cinema/controller/MovieSessionController.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        this.movieSessionService = movieSessionService;
+        this.movieSessionMapper = movieSessionMapper;
+    }
+
+    @PostMapping
+    public void add(@RequestBody MovieSessionRequestDto movieSessionDto) {
+        movieSessionService.add(movieSessionMapper.convertDtoToMovieSession(movieSessionDto));
+    }
+
+    @RequestMapping(value = "/available", method = RequestMethod.GET)
```
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/controller/MovieSessionController.java]: same here

**Review by RomanDubovskyi — COMMENTED** [review]
remember about informative pr names

File: src/main/java/com/dev/cinema/controller/MovieController.java (hunk 2 above)
Comment (MichaelShevchenko) [reply src/main/java/com/dev/cinema/controller/MovieController.java]: just wanted to try it out either ways
done

File: src/main/java/com/dev/cinema/controller/MovieSessionController.java (hunk 3 above)
Comment (MichaelShevchenko) [reply src/main/java/com/dev/cinema/controller/MovieSessionController.java]: done

File: src/main/java/com/dev/cinema/controller/MovieController.java (hunk 1 above)
Comment (MichaelShevchenko) [reply src/main/java/com/dev/cinema/controller/MovieController.java:22]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**